packages/tokamak/contracts-bedrock/scripts/start-deploy.sh

```diff
@@ -0,0 +1,215 @@
+}
+
+# checkScriptInput() {
+#   echo "Checking input"
+# }
+
+# checkAccount() {
+#   echo "Verifying accounts"
+# }
+
+installDependency() {
+  installGo
+  installJSPackages
+  installFoundry
+}
+
+installGo() {
+  echo "Install Go"
+  export PATH=$PATH:/usr/local/go/bin
+  if ! go version &> /dev/null; then
+    echo "Installing Go..."
+    cd /root
+    wget https://go.dev/dl/go1.21.13.linux-amd64.tar.gz
+    tar xvzf go1.21.13.linux-amd64.tar.gz
+    mv go /usr/local
+    export PATH=$PATH:/usr/local/go/bin
+    echo PATH=$PATH:/usr/local/go/bin >> ~/.bashrc
+    echo "   Installed $(go version)"
+    cd $currentPWD
+  else
+    echo "   Found $(go version)"
+  fi
+}
+
+installJSPackages() {
+  curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
+  DEBIAN_FRONTEND=noninteractive apt install -y nodejs
+  DEBIAN_FRONTEND=noninteractive apt install -y npm
+  npm install -g pnpm
+  curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | tee /usr/share/keyrings/yarnkey.gpg >/dev/null
+  echo "deb [signed-by=/usr/share/keyrings/yarnkey.gpg] https://dl.yarnpkg.com/debian stable main" | tee /etc/apt/sources.list.d/yarn.list
+  apt update && apt install yarn
+}
+
+installFoundry() {
+  echo "Install Foundry"
+  if ! forge --version &> /dev/null; then
+    curl -L https://foundry.paradigm.xyz | bash
+    mv /root/.foundry/bin/* /usr/bin/
+    source ~/.bashrc
+    foundryup
+    echo "export PATH=/root/.foundry/bin:$PATH" >> /root/.bashrc
+    source /root/.bashrc
+    export PATH=/root/.foundry/bin:$PATH
+    echo $PATH
+    echo "   Installed $(forge --version)"
+  else
+    echo "   Found $(forge --version)"
+  fi
+}
+
+buildSource() {
+  echo "Start buiding source code"
+  source ~/.bashrc
+  export PATH=$PATH:/usr/local/go/bin
+  cd $projectRoot
+  pnpm install
+  make submodules
+  make cannon-prestate
+  make op-node
+  cd $projectRoot/packages/tokamak/contracts-bedrock && pnpm build
+  cd $currentPWD
+}
+
+deployContracts() {
+  echo "Start deploying smart contracts"
+  echo $DEPLOY_CONFIG_PATH
+  export IMPL_SALT=$(openssl rand -hex 32)
+  cd $projectRoot/packages/tokamak/contracts-bedrock
+  forge script scripts/Deploy.s.sol:Deploy --private-key $GS_ADMIN_PRIVATE_KEY --broadcast --rpc-url $L1_RPC_URL --slow --legacy --non-interactive
+  cd $currentPWD
+}
+
+generateL2Genesis() {
+  echo "Generate L2 genesis"
+  deployResultFile=$projectRoot/packages/tokamak/contracts-bedrock/deployments/$(printf "%d-deploy.json" "$chainID")
+  cat $deployResultFile
+
+  export outdir=$projectRoot/build
+
+  if [ ! -d "$outdir" ]; then
+    mkdir -p "$outdir"
+    echo "Directory '$outdir' created."
+  else
+    echo "Directory '$outdir' already exists."
+    rm -rf $outdir/*
+  fi
+  cd $projectRoot
+  $projectRoot/op-node/bin/op-node genesis l2 \
+  --deploy-config $DEPLOY_CONFIG_PATH \
+  --l1-deployments $deployResultFile \
+  --outfile.l2 $outdir/genesis-$chainID.json \
+  --outfile.rollup $outdir/rollup-$chainID.json \
```

Comment:
Can you remove chainID in file name?
we need only genesis.json, rollup.json

Reply:
when uploading to s3 this files, if file name is rollup.json, genesis.json that it is fine user don't change the terraform config.